src/sections/layer_and_mask_information_section/layer.rs

```diff
@@ -107,12 +158,61 @@ impl PsdLayer {
         Ok(rgba)
     }
 
+    /// If layer represents a group, returns true
+    pub fn is_group(&self) -> bool {
+        self.sub_layers.is_some()
+    }
+
+    /// If layer represents a group, returns a vector with sub layers
+    pub fn layers(&self) -> Option<&PsdLayerGroup> {
+        self.sub_layers.as_ref()
+    }
+
     // Get one of the PsdLayerChannels of this PsdLayer
     fn get_channel(&self, channel: PsdChannelKind) -> Option<&ChannelBytes> {
         self.channels.get(&channel)
     }
 }
 
+/// Section divider constants.
+/// There are 4 possible values:
+///
+/// 0 = any other type of layer
+const DIVIDER_ANY_OTHER_TYPE: i32 = 0;
+/// 1 = open "folder"
+const DIVIDER_OPEN_FOLDER: i32 = 1;
+/// 2 = closed "folder"
+const DIVIDER_CLOSED_FOLDER: i32 = 2;
+/// 3 = bounding section divider, hidden in the Photoshop UI
+const DIVIDER_BOUNDING_SECTION: i32 = 3;
+
+/// GroupDivider represents tag type of Section divider.
+#[derive(Debug)]
+pub(super) enum GroupDivider {
+    /// Any other type divider
+    Other,
+    /// Open folder divider
+    OpenFolder,
+    /// Close folder divider
+    CloseFolder,
+    /// Bounding section divider
+    BoundingSection,
+    /// Not a divider at all
+    NotDivider,
+}
+
+impl GroupDivider {
+    pub(super) fn match_divider(divider: i32) -> GroupDivider {
+        match divider {
+            DIVIDER_ANY_OTHER_TYPE => GroupDivider::Other,
+            DIVIDER_OPEN_FOLDER => GroupDivider::OpenFolder,
+            DIVIDER_CLOSED_FOLDER => GroupDivider::CloseFolder,
+            DIVIDER_BOUNDING_SECTION => GroupDivider::BoundingSection,
+            _ => GroupDivider::NotDivider
```

Comment:
Similarly here - we have a type `GroupDivider` that has the ability to not be a divider. More idiomatic to just have an option (a noted below)